docs/GettingStarted.md

```diff
@@ -123,8 +122,16 @@ architecture (powerpc or amd64) from the website and run it.
         ...
 
 
+Note: 
+	1)If there is diffrent OS running on the swicth already.Then stop it at U-boot mode,
+	      => printenv
+           check the onie details.Open the ONIE in rescue mode by,
+	      => run onie_rescue
+           It will take you to the ONIE environment.
 
-
+       2) To load ONL installer from ONIE directly from the devlopement environment ,Just run a http server directly from the location (example:python -m SimpleHTTPServer 8000) and access it as,
```

Comment:
please fix space and 'J'.  Also mention that you can install via scp with two steps, e.g., 

ONIE:/ # scp buildmachine:/path/to/onl-c82f711-powerpc-all.2016.01.22.01.25.installer ONL.installer # update for specific file/date/build
ONIE:/ # sh ONL.installer